packages/integration-sdk-runtime/src/synchronization/index.ts

```diff
@@ -558,7 +535,6 @@ export async function uploadData<T extends UploadDataLookup, K extends keyof T>(
   { job, apiClient, logger }: SynchronizationJobContext,
   type: K,
   data: T[K][],
-  uploadBatchSize?: number,
   uploadBatchSizeInBytes?: number,
```

Comment:
the `uploadBatchSize` argument here was unused, which means all of the above is unused.

Reply:
A follow up to this PR will allow overriding `DEFAULT_UPLOAD_BATCH_SIZE_IN_BYTES = 5_000_000;` to be higher.

Reply:
https://github.com/JupiterOne/sdk/pull/967